deployment/network-operator/README.md

```diff
@@ -302,16 +302,17 @@ We have introduced the following Chart parameters.
 
 ### General parameters
 
-| Name | Type | Default | description |
-| ---- | ---- | ------- | ----------- |
-| `nfd.enabled` | bool | `True` | deploy Node Feature Discovery |
-| `sriovNetworkOperator.enabled` | bool | `False` | deploy SR-IOV Network Operator |
-| `psp.enabled` | bool | `False` | deploy Pod Security Policy |
-| `operator.repository` | string | `nvcr.io/nvidia/cloud-native` | Network Operator image repository |
-| `operator.image` | string | `network-operator` | Network Operator image name |
-| `operator.tag` | string | `None` | Network Operator image tag, if `None`, then the Chart's `appVersion` will be used |
-| `operator.imagePullSecrets` | list | `[]` | An optional list of references to secrets to use for pulling any of the Network Operator image |
-| `deployCR` | bool | `false` | Deploy `NicClusterPolicy` custom resource according to provided parameters |
+| Name | Type | Default | description                                                                                                          |
+| ---- | ---- | ------- |----------------------------------------------------------------------------------------------------------------------|
+| `nfd.enabled` | bool | `True` | deploy Node Feature Discovery                                                                                        |
+| `sriovNetworkOperator.enabled` | bool | `False` | deploy SR-IOV Network Operator                                                                                       |
+| `psp.enabled` | bool | `False` | deploy Pod Security Policy                                                                                           |
+| `imagePullSecrets` | list | `[]` | An optional list of references to secrets to use for pulling any of the Network Operator image if it's not overrided |
```

Comment:
can you also add a short example in the description ?

Reply:
will add it asap